content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -0,0 +1,3253 @@
+- For each `User` record, returns the `title` field of all nested `Post` records sorted by `title`
+
+<CodeWithResult>
+
+<Cmd>
+
+```ts
+const usersWithPosts = await prisma.user.findMany({
+  orderBy: {
+    email: 'asc',
+  },
+  include: {
+    posts: {
+      select: {
+        title: true,
+      },
+      orderBy: {
+        title: 'asc',
+      },
+    },
+  },
+})
+```
+
+</Cmd>
+
+<CmdResult>
+
+```json
+[
+  {
+    "id": 2,
+    "email": "alice@prisma.io",
+    "name": "Alice",
+    "posts": [
+      {
+        "title": "Watch the talks from Prisma Day 2019"
+      }
+    ]
+  },
+  {
+    "id": 3,
+    "email": "ariadne@prisma.io",
+    "name": "Ariadne",
+    "posts": [
+      {
+        "title": "How to connect to a SQLite database"
+      },
+      {
+        "title": "My first day at Prisma"
+      }
+    ]
+  },
+  {
+    "id": 1,
+    "email": "bob@prisma.io",
+    "name": "Bob",
+    "posts": [
+      {
+        "title": "Follow Prisma on Twitter"
+      },
+      {
+        "title": "Subscribe to GraphQL Weekly for community news "
+      }
+    ]
+  }
+]
+```
+
+</CmdResult>
+
+</CodeWithResult>
+
+##### Sort one user's nested list of `Post` records
+
+The following example retrieves a single `User` record by ID, as well as a list of nested `Post` records sorted by `title`:
+
+<CodeWithResult>
+
+<Cmd>
+
+```ts
+const userWithPosts = await prisma.user.findUnique({
+  where: {
+    id: 1,
+  },
+  include: {
+    posts: {
+      orderBy: {
+        title: 'desc',
+      },
+      select: {
+        title: true,
+        published: true,
+      },
+    },
+  },
+})
+```
+
+</Cmd>
+
+<CmdResult>
+
+```json
+{
+  "email": "sarah@prisma.io",
+  "id": 1,
+  "name": "Sarah",
+  "extendedProfile": null,
+  "role": "USER",
+  "posts": [
+    {
+      "title": "Prisma Day 2020",
+      "published": false
+    },
+    {
+      "title": "My first post",
+      "published": false
+    },
+    {
+      "title": "All about databases",
+      "published": true
+    }
+  ]
+}
+```
+
+</CmdResult>
+
+</CodeWithResult>
+
+##### Sort by `enum`
+
+The following sorts all `User` records by `role` (an `enum`):
+
+<CodeWithResult>
+
+<Cmd>
+
+```ts
+const sort = await prisma.user.findMany({
+  orderBy: {
+    role: 'desc',
+  },
+  select: {
+    email: true,
+    role: true,
+  },
+})
+```
+
+</Cmd>
+
+<CmdResult>
+
+```json
+[
+  {
+    "email": "emma@prisma.io",
+
+    "role": "USER"
+  },
+  {
+    "email": "suma@prisma.io",
+    "role": "ADMIN"
+  },
+  {
+    "email": "kwame@prisma.io",
+    "role": "ADMIN"
+  },
+  {
+    "email": "pearl@prisma.io",
+    "role": "ADMIN"
+  }
+]
+```
+
+</CmdResult>
+
+</CodeWithResult>
+
+### <inlinecode>distinct</inlinecode>
+
+See also: [Aggregation, grouping, and summarizing](../../../concepts/components/prisma-client/aggregation-grouping-summarizing#select-distinct) <span class="concept"></span>
+
+#### Reference
+
+```ts
+distinct?: Enumerable<UserDistinctFieldEnum>
+```
+
+```ts
```

Comment:
I noticed that you're not using the `index.d.ts` file annotation here. We should probably be consistent with this across the entire page.